### PR TITLE
Feat/refactor properties components

### PR DESCRIPTION
**Pull request by vvandijck**
This pull request includes:
- Cleaned up parts of `PropertiesList` and `PropertiesItem` components. Including styling.
- Use `PropertiesList` and `PropertiesItem` as much as possible to reduce the number of different "lists" of properties and data.
- Views are now more identical between the chrome extension and the web app.

### CODE REVIEW

File: projects/browser-extension/vue-app/src/views/favicon.view.vue [hunk 1]
```diff
@@ -56,12 +59,14 @@ export default {
...
     ExternalLink,
     PanelSection,
+    PropertiesItem,
     PropertiesList,
     WarningIcon,
   },
   computed: {
     ...mapState([ 'head' ]),
     favicons() {
+      console.log(findFavicons(this.head))
```
Comment (Jamerrone) [inline projects/browser-extension/vue-app/src/views/favicon.view.vue]: Can be removed, right?

File: projects/web-app/src/components/input-url/input-url.vue [hunk 2]
```diff
@@ -18,7 +18,7 @@ import useHead from '@/composables/use-head';
 
 export default {
   setup: () => {
-    const url = ref('https://www.voorhoede.nl/en/');
+    const url = ref('https://www.npmjs.com/');
```
Comment (Jamerrone) [inline projects/web-app/src/components/input-url/input-url.vue]: Waarom de npm website?

**Review by Jamerrone — APPROVED** [review]
1 question, and one console.log that needs to be removed. Can be merged afterwards.

File: projects/browser-extension/vue-app/src/views/favicon.view.vue (hunk 1 above)
Comment (vvandijck) [reply projects/browser-extension/vue-app/src/views/favicon.view.vue]: Yes 👍

File: projects/web-app/src/components/input-url/input-url.vue (hunk 2 above)
Comment (vvandijck) [reply projects/web-app/src/components/input-url/input-url.vue]: I used this to test, i'll remove it.

**Pull request merged**